packages/node/src/lib/novu.ts

```diff
@@ -24,28 +22,22 @@ export class Novu extends EventEmitter implements INovu {
     });
 
     this.subscribers = new Subscribers(this.http);
+    this.events = new Events(this.http);
+    this.trigger = this.events.trigger;
+    this.broadcast = this.events.broadcast;
   }
 
-  async trigger(eventId: string, data: ITriggerPayloadOptions) {
-    return await this.http.post(`/events/trigger`, {
-      name: eventId,
-      to: data.to,
-      payload: {
-        ...data?.payload,
-      },
-      overrides: data.overrides || {},
-    });
-  }
-
-  async broadcast(eventId: string, data: IBroadcastPayloadOptions) {
-    return await this.http.post(`/events/trigger/broadcast`, {
-      name: eventId,
-      payload: {
-        ...data?.payload,
-      },
-      overrides: data.overrides || {},
-    });
-  }
+  // TODO: remove this in the a future major version
+  /**
+   * @deprecated Use {@link Events.trigger} instead.
+   */
+  public trigger: typeof Events.prototype.trigger;
+
+  // TODO: remove this in the a future major version
+  /**
+   * @deprecated Use {@link Events.trigger} instead.
+   */
+  public broadcast: typeof Events.prototype.broadcast;
```

Comment:
Could we please restore this functions to be at the top level of the Novu library? Because this is the most common usecase for the SDK I would prefer if we leave it as it was before to have the "trigger" and "broadcast" endpoints to be used from `novu.trigger` instead of `novu.events.trigger` I think that the API looks a bit more clean without the "events" for those 2 methods. 

All the other events methods can leave under the events namespace. 

You can use behind the scenes the method in Events, but just in terms of public API would prefer to have it from the root

Reply:
So should I keep what I've done but just remove the deprecation notices?

Reply:
Yes, removing the comments should work perfectly!